view/page/linux.php

```diff
@@ -7,8 +7,7 @@
     <div class="span6">
       <h3>For the Efficient and Lazy</h3>
       <ol>
-        <li>Make a folder called <code>lbry</code> where you want everything to reside.</li>
-        <li>Download and run <a href="https://raw.githubusercontent.com/lbryio/lbry-setup/master/lbry_setup.sh" class="link-primary">this script</a> from that folder.</li>
+        <li><code>(mkdir lbry; cd lbry; wget https://raw.githubusercontent.com/lbryio/lbry-setup/master/lbry_setup.sh; chmod 755 lbry_setup.sh; ./lbry_setup.sh; bin/lbry/lbrycrdd -server -daemon)</code></li>
```

Comment:
This should all be good except for the last command. I don't think we want the installer to also run lbrycrdd. lbrynet-console and lbrynet-gui launch lbrycrdd when they start up and, if they launched it, shut it down when they exit. That way it's only running if the user wants it to be running. (Also, the path to the binary will be bin/lbrycrdd.) Thoughts?